migrations/sql/V2024.12.11.22.33__add_user_version_history_table_backfill.sql

```diff
@@ -0,0 +1,6 @@
+-- This file was generated by the generate_history_table_ddl command
+-- The file contains the data migration to backfill history records for the {table} table
+with transaction AS (insert into transaction(id) values(DEFAULT) RETURNING id)
+insert into user_version (transaction_id, operation_type, end_transaction_id, "create_user", "create_timestamp", "update_user", "update_timestamp", "deleted_ind", "sub", "email", "given_name", "family_name", "display_name", "idir_username", "identity_provider", "idir_user_guid")
+select t.id, '0', null, "create_user", "create_timestamp", "update_user", "update_timestamp", "deleted_ind", "sub", "email", "given_name", "family_name", "display_name", "idir_username", "identity_provider", "idir_user_guid"
+from "user",transaction t;
```

Comment:
For future me/us- does omitting `last_logged_in` from here prevent new records from being created when there's a new `last_logged_in`?

Reply:
It exludes last_logged_in from the versioning altogether, so it doesn't trigger a version when it's updated and the versions don't even have that field.  When it's fetched, it grabs the value from the non-versioned user entity.